pigar/__main__.py

```diff
@@ -197,16 +203,18 @@ def extract_reqs(self):
         logger.info('Finish local environment checking.')
         return reqs, try_imports, guess
 
-    def _write_reqs(self, reqs):
+    def _write_reqs(self, reqs, minimal=False):
+        """Write requirements to a file"""
         print(Color.BLUE('Writing requirements to "{0}"'.format(
             self._save_path)))
         with open(self._save_path, 'w+') as f:
             f.write('# Requirements automatically generated by pigar.\n'
                     '# https://github.com/damnever/pigar\n')
```

Comment:
I think here needs an extra `f.write('\n')` if `not minimal`.

Reply:
That makes sense. I will add it.